docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
```

Comment:
It sounds like we should add this navigation case in https://master--material-ui.netlify.app/experiments/docs/installation/, so that when we make changes, we can also cover nested navigation.

Reply:
@cherniavskii sweet, I'm glad you enjoyed it, and thanks for calling attention to this ⎯ @oliviertassinari had already signaled it! Is there a way I can play with this to try to tackle these issues? I don't know how I'd change something on this repo and have an effect on the X docs 🤔 Having a mocked scenario on the experiments folder would 100% help, I think!

Either way, apologies for not having consulted y'all before merging that PR, that's my bad 😞

Reply:
I can add use cases from MUI X to https://master--material-ui.netlify.app/experiments/docs/installation/ 👍

Reply:
@danilo-leal Here you go: https://github.com/mui/material-ui/pull/38047